governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Template - [Executive Vote] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023
+summary: Scope-defined parameter changes; Spark Protocol D3M parameter changes; set Fortunafi Debt Ceiling to zero; approve DAO Resolution for HV Bank to terminate future purchases; Aligned Delegate compensation for August 2023; trigger Spark Protocol Proxy Spell; Core Unit MKR vesting transfers; scuttle MCD_CAT contract.  
+date: 2023-09-13T00:00:00.000Z
+address: "$spell_address"
+
+
+
+
+
+---
+
+# [Executive Proposal] Scope-Defined Parameter Changes, Spark Protocol D3M Parameter Changes, Set Fortunafi Debt Ceiling to Zero, DAO Resolution for HV Bank, Delegate Compensation and Other Actions - September 13, 2023
+
+The Reserve Governance Facilitators, Sidestream, Dewiz and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
```

Comment:
I'll omit my comments about the order, but can we organize these into coherent groups to enhance readability? Perhaps something like this:

- Parameter Modifications
- Updates on RWA
- Expenses
- Administrative Updates

I believe that if we're avoiding the use of "changes" or "additions" as GovAlpha used to do, we should introduce another way to prevent the executive summary from becoming a lengthy text block.

Reply:
Thanks JS for the suggestion!

After thinking on it, I don't think grouping items into sections/classes is wise in the Executive Summary. 

Here's the pros:  I tried drafting a Exec Summary v2 to see visually how it looks. It offers a clearer signal to readers in terms of similarities between items, and also gives a brief takeaway about the essence of each proposed item.

<img width="1019" alt="Screenshot 2023-09-09 at 3 27 13 PM" src="https://github.com/makerdao/community/assets/139828768/80b354de-6ba6-4e26-9129-3ad200ee8f2b">


But the good is outweighed I think by the cons. In our training, we were advised to "Order the items by judged impact/importance."

Items that are placed together into a single class do not necessarily have the same overall impact/importance in relation to the rest of the remaining items in the proposal. For instance, the 2 Spark Protocol-related Actions, which are grouped together in the Exec Summary v2 do not have the same judged impact/importance vis a vis the rest of the items. 

See this previous [Exec Vote](https://github.com/makerdao/community/blob/ca6fe657cccd887ac12a0ec15f80d412efa28b06/governance/votes/Executive%20Vote%20-%20August%202%2C%202023.md), where the Spark D3M parameter adjustments were listed much higher in the Copy than the Spark Proxy Spell, due to difference in impact. And in the immediate instance, I think the Spark Protocol parameter changes are more minor in nature. 

Another example:  2 RWA vault updates that are grouped together in the Exec Summary simply bc they share the same nature may be very different in terms of judged impact/importance. For example, one such update may be a complete restructuring of the vault (New Silver). While another vault update may involve a simple DAO resolution or a doc parameter update.

So if we went with the v.2 of Exec Summary, we would have 2 choices:

1.  change ordering of items in the **Proposal Details** section, so that the ordering matches that of the Executive Summary. We would thus break the principle cited above of ordering items by judged impact.

2.  On the other hand, we can choose to order items in **Proposal Details** section so as to adhere to the principle cited above. But in this case, the ordering of items in the above **Executive Summary** would not match the ordering of items in the below **Proposal Details** section. This would be v confusing to readers, I think, as the Executive Summary is meant to be a "table of contents" that is legible and convenient. 

I think it's best for the executive summary bullet points to match the order of the items in the below Proposal Details section. 

I agree that readability is key; and I think we have made significant improvements to breaking up the blocks of text by introducing the provenance preface for each item

As long as we keep the sentences in the Executive Summary relatively succinct, the Executive Summary will not be prohibitive in appearance/readability. 

In sum, I think in this instance, we should keep to current practice, especially for this first Exec Vote. 

@VoteWizard feel free to chime in with your thoughts as well. We can reconsider and implement at another date.

Reply:
Thank you for the comprehensive discussion. I've read both perspectives and find merit in them.

However, I tend to agree with the guidelines we've previously adopted as per GovAlpha's practices. The principle of ordering items by their "judged impact/importance" makes a lot of sense and helps to direct attention to the most critical elements of a proposal.

The idea of grouping items into coherent categories like "Parameter Modifications," "Updates on RWA," "Expenses," and "Administrative Updates" has its merits in terms of readability, but as @Le-Bateleur-2023  pointed out, this could lead to a disconnect between the Executive Summary and the Proposal Details in terms of their order.